tests/fixtures/ead/ead_record_all_fields.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<records>
+    <record xmlns="http://www.openarchives.org/OAI/2.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+        <header>
+            <identifier>oai:mit//repositories/2/resources/1</identifier>
+            <datestamp>2021-11-06T14:35:55Z</datestamp>
+        </header>
+        <metadata>
+            <ead xmlns="urn:isbn:1-931666-22-9" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:isbn:1-931666-22-9 http://www.loc.gov/ead/ead.xsd">
+                <eadheader countryencoding="iso3166-1" dateencoding="iso8601" findaidstatus="completed" langencoding="iso639-2b" repositoryencoding="iso15511">
+                    <eadid countrycode="US" mainagencycode="US-mcm">VC-0002</eadid>
+                    <filedesc>
+                        <titlestmt>
+                            <titleproper>
+                                Guide to the Charles J. Connick Stained Glass Foundation Collection
+                                <num>VC.0002</num>
+                            </titleproper>
+                            <titleproper>
+                                Title 2
+                                <num>VC.0002</num>
+                            </titleproper>
+                            <titleproper>
+                                Title 3
+                            </titleproper>
+                            <titleproper></titleproper>
```

Comment:
Not necessary now but when you start on the next block of work I'd move this empty field to the missing fields fixture.

Reply:
👍 That was the plan